site/content/en/references/kustomize/cmd/localize/_index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "kustomize localize"
+linkTitle: "localize"
+type: docs
+weight: 9
+description: >
+    Replace urls with local paths
+---
+
+Disclaimer: This is an alpha command. Please see the [command proposal](https://github.com/kubernetes-sigs/kustomize/blob/master/proposals/22-04-localize-command.md)
+for full capabilities.
+
+### Feedback
+
+Please leave your feedback for this command under [the following issue](https://github.com/kubernetes-sigs/kustomize/issues/4996).
+
+### Man
+
+The `kustomize localize` command makes a recursive copy of a kustomization
+in which referenced urls are replaced by local paths to their downloaded content.
+
+The purpose of this command is to create a copy on which
+`kustomize build`<sup>[[build]](#notes)</sup> produces the same output
+without a network connection.
+
+The command takes the following form:
+
+<pre>
+<b>kustomize localize</b> [<ins>target</ins> [<ins>newDir</ins>]] [--scope <ins>scope</ins>]
+</pre>
+
+where
+
+* `target` is the [kustomization directory](https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#kustomization-root) 
+to localize. This value can be a local path or a [remote root](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md). 
+The default value is the current working directory.
+* `newDir` is the destination of the "localized" copy that the command
+will create. The destination must reside in an existing directory.
```

Comment:
"The destination must reside in an existing directory." suggest replacing this line with the answers about:
1. if the `newDir` does not exist, will `kustomize localize` create it ? 
2. if the `newDir` exists, does it have to be empty?
3. (if answer to 2 is No) what happens when conflicts, will `kustomize localize` detects that?

Reply:
Done. I updated the description here: https://github.com/kubernetes-sigs/cli-experimental/pull/313/files#diff-3de085d06ca2f9e7f7dbd1c0c3c1f86786e2555ac850003aae4ab3875249b8d5R41-R43

Let me know if it's still unclear.